t/24-worker-jobs.t

```diff
@@ -95,7 +95,7 @@ sub wait_until_uploading_logs_and_assets_concluded {
     has pool_directory  => undef;
 }
 {
-    package Test::FakeClient;
+    package Test::FakeClient;                                                          # uncoverable statement count:2
```

Comment:
Just a note: That this and similar lines aren't accounted it likely a limitation of the coverage tracking.

Reply:
Yes, I agree :)